model/src/main/kotlin/Package.kt

```diff
@@ -75,8 +75,9 @@ data class Package(
     val declaredLicensesProcessed: ProcessedDeclaredLicense = DeclaredLicenseProcessor.process(declaredLicenses),
 
     /**
-     * The concluded license as an [SpdxExpression]. It can be used to correct the license of a package in case the
-     * [declaredLicenses] found in the packages metadata or the licenses detected by a scanner do not match reality.
+     * The concluded license as an [SpdxExpression]. It can be used to override the [declared licenses]
+     * [Package.declaredLicenses] and / or the [detected license expression][LicenseFinding.license] of a package in
```

Comment:
* `Package.` is not required for a property of the same class.
* I think just "and" instead of "and / or" is good enough, the "or" could be confusing. Also, why not "detected licenses" instead of "detected license expression" (as it can be multiple)? (same below)

Reply:
> Also, why not "detected licenses" instead of "detected license expression" (as it can be multiple)?

I wanted to emphasize the different data types: While `declaredLicenses` is a set of strings (where plural IMO fits better), `LicenseFinding.license` is a single SPDX expression.

Reply:
Still, in the context of addressing [@fviernau's comment](https://github.com/oss-review-toolkit/ort/pull/5686#discussion_r954311592), I went for a yet simpler sentence that uses plural for both license origins.